src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -11699,1170 +11730,1462 @@ namespace OutputReportTabular {
-								// short wave is already accumulated by surface
-								lightSWConvFromSurf += lightSWRadSeq( desSelected, kTimeStep - mStepBack + 1, jSurf ) * decayCurveCool( mStepBack, jSurf );
-								feneSolarConvFromSurf += feneSolarRadSeq( desSelected, kTimeStep - mStepBack + 1, jSurf ) * decayCurveCool( mStepBack, jSurf );
-							} // for mStepBack
-							peopleConvIntoZone += peopleConvFromSurf;
-							equipConvIntoZone += equipConvFromSurf;
-							hvacLossConvIntoZone += hvacLossConvFromSurf;
-							powerGenConvIntoZone += powerGenConvFromSurf;
-							lightLWConvIntoZone += lightLWConvFromSurf;
-							lightSWConvIntoZone += lightSWConvFromSurf;
-							feneSolarConvIntoZone += feneSolarConvFromSurf;
-							// determine the remaining convective heat from the surfaces that are not based
-							// on any of these other loads
-							//negative because heat from surface should be positive
-							surfDelaySeqCool( kTimeStep, jSurf ) = -loadConvectedNormal( desSelected, kTimeStep, jSurf ) - netSurfRadSeq( desSelected, kTimeStep, jSurf ) - ( peopleConvFromSurf + equipConvFromSurf + hvacLossConvFromSurf + powerGenConvFromSurf + lightLWConvFromSurf + lightSWConvFromSurf + feneSolarConvFromSurf ); //remove net radiant for the surface
-							// also remove the net radiant component on the instanteous conduction for fenestration
-							if ( Surface( jSurf ).Class == SurfaceClass_Window ) {
-								adjFeneSurfNetRadSeq += netSurfRadSeq( desSelected, kTimeStep, jSurf );
-							}
-						} // for jSurf
-						peopleDelaySeqCool( kTimeStep ) = peopleConvIntoZone;
-						equipDelaySeqCool( kTimeStep ) = equipConvIntoZone;
-						hvacLossDelaySeqCool( kTimeStep ) = hvacLossConvIntoZone;
-						powerGenDelaySeqCool( kTimeStep ) = powerGenConvIntoZone;
-						//combine short wave (visible) and long wave (thermal) impacts
-						lightDelaySeqCool( kTimeStep ) = lightLWConvIntoZone + lightSWConvIntoZone;
-						feneSolarDelaySeqCool( kTimeStep ) = feneSolarConvIntoZone;
-						// also remove the net radiant component on the instanteous conduction for fenestration
-						feneCondInstantSeq( desSelected, kTimeStep, iZone ) -= adjFeneSurfNetRadSeq;
-					} // for kTimeStep
-				} // if desSelected != 0
-
-			} // if allocated( CalcFinalZoneSizing )
-
-			//---- Cooling Peak Load Components Sub-Table
-			WriteReportHeaders( "Zone Component Load Summary", Zone( iZone ).Name, isAverage );
-
-			rowHead.allocate( rGrdTot );
-			columnHead.allocate( cPerc );
-			columnWidth.dimension( cPerc, 14 ); //array assignment - same for all columns
-			tableBody.allocate( cPerc, rGrdTot );
-
-			if ( unitsStyle != unitsStyleInchPound ) {
-				columnHead( cSensInst ) = "Sensible - Instant [W]";
-				columnHead( cSensDelay ) = "Sensible - Delayed [W]";
-				columnHead( cSensRA ) = "Sensible - Return Air [W]";
-				columnHead( cLatent ) = "Latent [W]";
-				columnHead( cTotal ) = "Total [W]";
-				columnHead( cPerc ) = "%Grand Total";
+			Array2D< Real64 > decayCurve;
```

Comment:
I assume that this will create a full copy and not just create a reference.  I don't think it will matter for your work here either way, so I'm carrying on...